include/deal.II/base/utilities.h

```diff
@@ -429,12 +429,27 @@ namespace Utilities
 
   /**
    * Given an arbitrary object of type T, use boost::serialization utilities
-   * to pack the object into a vector of characters. The object can be unpacked
-   * using the Utilities::unpack function below.
+   * to pack the object into a vector of characters and append it to the
+   * given buffer. The number of elements that have been added to the buffer
+   * will be returned. The object can be unpacked using the Utilities::unpack
+   * function below.
    *
    * If the library has been compiled with ZLIB enabled, then the output buffer
    * is compressed.
    *
+   * If many consecutive calls with the same buffer are considered, it is
+   * recommended for reasons of performance to ensure that its capacity is
+   * sufficient.
+   *
```

Comment:
please document what this function returns

Reply:
Are lines 433/434 sufficient? If not, I"ll provide an extra paragraph.

Reply:
ah, sure, i did not notice this.